content/doc/book/installing/_kubernetes.adoc

```diff
@@ -552,3 +553,183 @@ This may also be found at:
 ----
 
 You have successfully installed Jenkins on your Kubernetes cluster and can use it to create new and efficient development pipelines.
+
+==== Install Jenkins with Jenkins Operator
+
+The Jenkins Operator is a Kubernetes native Operator which manages operations
+for Jenkins on Kubernetes. 
+It was built with immutability and declarative configuration as code in mind. 
+The Jenkins Operator is easy to install with just a few manifest and allows 
+users to configure and manage Jenkins on Kubernetes.
+
+===== Prerequisites
+
+Jenkins Operator::
+If you don't have Jenkins Operator installed and configured locally, 
+see the sections below to <<Install Jenkins Operator>>.
+
+===== Install Jenkins Operator
+
+Requirements::
+
+To run Jenkins Operator, you will need:
+
+. Access to a Kubernetes cluster. If you don't have a running Kubernetes cluster, 
+see the link:/doc/book/installing/kubernetes/#create-a-kubernetes-cluster-with-minikube[Create a Kubernetes cluster with minikube] section above.
+
+. kubectl version 1.11+
+
+===== Configure Custom Resource Definition
+
+The Custom Resource Definition (CRD) API has been introduced to Kubernetes in v1.7 
+and it enables users to add custom APIs to their Kubernetes cluster which can be
+used like any other native Kubernetes objects.
+Defining a CRD object creates a new custom resource with a name and schema that you specify. The Kubernetes API serves and handles the storage of your custom resource.
+
+Install Jenkins Custom Resource Definition::
+
+[source,bash]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/crds/jenkins
```

Comment:
Might become a problem when/if the master branch is renamed

Reply:
Would it be okay if I put the yaml files for the guide in the jenkins.io repository? That way we can better track changes.

Reply:
I do not mind. I would rather use a separate repo, but I do not feel strongly about it

Reply:
better not to do that, it's very common for official deploy instructions to be apply these yaml files in the official repo, no one is going to maintain them in this repo.

Reply:
@markyjackson-taulia recommends that we fix this URL so that it points to a location that exists and that it includes a comment warning that we need to confirm this works periodically.